src/platforms/GitHub/index.ts

```diff
@@ -75,7 +75,7 @@ export function isEnterprise() {
        * </a>
        */
       $(
-        'a.Header-link[aria-label="Homepage Enterprise"]',
+        'a.Header-link[aria-label^="Homepage"]',
```

Comment:
```suggestion
        [
          'a.Header-link[aria-label="Homepage Enterprise"]',
          'a.Header-link[aria-label="Homepage"]', // legacy support
        ].join(),
```

This would prevent matching unexpected results like something has `aria-label="Homepage xxx"`.